gui/public/i18n/en/translation.ftl

```diff
@@ -522,6 +525,16 @@ onboarding-connect_tracker-connected_trackers = { $amount ->
 } connected
 onboarding-connect_tracker-next = I connected all my trackers
 
+## Tracker calibration tutorial
+onboarding-calibration_tutorial = IMU Calibration Tutorial
+onboarding-calibration_tutorial-subtitle = This will help with tracker drifting!
+onboarding-calibration_tutorial-description = When you turn on your trackers, they need to stay for a moment on a flat surface to calibrate. Let's try doing the same thing by clicking the Calibrate button, <b>try to not move them!</b>
+onboarding-calibration_tutorial-calibrate = Calibrate
+onboarding-calibration_tutorial-status-waiting = Waiting for you
+onboarding-calibration_tutorial-status-calibrating = Calibrating
+onboarding-calibration_tutorial-status-success = You did it!
```

Comment:
```suggestion
onboarding-calibration_tutorial-status-success = Done!
```

Reply:
:sob:

Reply:
i just want non-corporative stuff, its a tutorial!!!

Reply:
It’s not about sounding like a corporation. It’s about consistency, making localization simpler, and not *obviously* treating the user like a dummy.